Project2015To2017/Reading/Conditionals/OperatorExpressionNode.cs

```diff
@@ -8,7 +8,7 @@ namespace Project2015To2017.Reading.Conditionals
     /// <summary>
     /// Base class for nodes that are operators (have children in the parse tree)
     /// </summary>
-    internal abstract class OperatorExpressionNode : GenericExpressionNode
+    public abstract class OperatorExpressionNode : GenericExpressionNode
```

Comment:
Yeah, that's necessary due to `ConditionEvaluationStateImpl` also becoming public and containing instances of this class.